Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -105,7 +105,9 @@ def use_correct_linux_tap(formula, args:)
         end
       end
     end
-    [formula.tap&.full_name, "origin/master", "-"]
+    origin_branch = Utils.popen_read("git", "-C", formula.tap.path.to_s, "symbolic-ref", "-q", "--short",
+                                     "refs/remotes/origin/HEAD").chomp
```

Comment:
```suggestion
    origin_branch = Utils.popen_read("git", "-C", formula.tap.path.to_s, "symbolic-ref", "-q", "--short",
                                     "refs/remotes/origin/HEAD").chomp.presence
    origin_branch ||= "origin/master"
```

There's some circumstances in which this won't be set.